docs/src/resampling.md

```diff
@@ -67,10 +67,18 @@ The right image is the upsampled version of the left one.
 ![](assets/image_low_res.png)
 ![](assets/image_high_res.png)
 
+There are fast versions (`upsample2`)for upsampling by a factor of 2.
 
 
 ## Function References
 ```@docs
 FourierTools.resample
+FourierTools.resample_by_FFT
+FourierTools.resample_by_RFFT
+FourierTools.resample_nfft
+FourierTools.resample_czt
 FourierTools.upsample2
+FourierTools.upsample2_abs2
+FourierTools.upsample2_1D
+FourierTools.barrel_pin
 ```
```

Comment:
I'm actually against export all those functions. Some of them are rather internally (such as `resample_by_RFFT`).

So I would only export `upsample2` and `resample`

Reply:
OK

Reply:
Why not "barrel_pin"? This is not internal.

Reply:
Ok for `barrel_pin`.

But rename it to something more expressive.

Reply:
I left barrel_pin in for export

Reply:
What about changing the name?

Reply:
backward compatibility? What is wrong with the name?

Reply:
Yes, I was confused why GitHub indicated change of these lines.
Then we leave it but nevertheless.

How should one know that `pin` stands to `pincushion` and not for `to pin`.

Mainly it's a distortion, so something like: `distort_radial` would be more expressive
https://en.wikipedia.org/wiki/Distortion_(optics)#Radial_distortion

Reply:
we should leave it as is. 'pincushion' is described in the help and I find the name OK. Also we should not change it due to backwards compatibility issues.